src/pages/Anix/main.ts

```diff
@@ -0,0 +1,127 @@
+import { Animeworld } from '../Animeworld/main';
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+
+export const Anix: pageInterface = {
+  domain: 'https://anix.to',
+  languages: ['English'],
+  name: 'Anix',
+  type: 'anime',
+  isSyncPage(url: string): boolean {
+    return true;
+  },
+  sync: {
+    getTitle(url: string): string {
+      return j.$('h1.anime-name').text().trim();
+    },
+    getIdentifier(url: string): string {
+      const anime = url.split('/')[4];
+      const animeWords = anime.split('-');
+      return animeWords[animeWords.length - 1];
+    },
+    getOverviewUrl(url: string): string {
+      return utils.absoluteLink(
+        j.$('div.range-wrap > div.range > div > a').first().attr('href'),
```

Comment:
Selector too specific and would break with any change

Reply:
Changed to `div.range a`